packages/containers/src/SidePanel/SidePanel.container.js

```diff
@@ -1,22 +1,25 @@
 import get from 'lodash/get';
 import React, { PropTypes } from 'react';
 import { SidePanel as Component } from 'react-talend-components';
-import { api, componentState } from 'react-cmf';
+import { componentState } from 'react-cmf';
 import { Map } from 'immutable';
 
+import { getActionsProps } from '../actionAPI';
+
 export const DEFAULT_STATE = new Map({
 	docked: false,
 });
 
-
-export function getActions(actionIds, context) {
+export function getActions(actionIds, context, model) {
```

Comment:
This means that `const actions = getActions(actionIds, this.context);` will fail because it doesn't provide it.